### PR TITLE
Cleanup on Display labels & uris

### PR DESCRIPTION
**Pull request by lamtu1**
`UPDATE:` Continue to update more on the display usage of the new `label$uri`

`TASKS:` To-Do lists

- [x] Update the `data_source.html.erb` to not use the `zipped_values` method and use the new `label$uri` method
- [x] Update the `OregonDigital::MetadataDownload#metadata_row` to get the new metadata without doing much fetching and cleanup as well
- [x] Update the `OregonDigital::PresentsDataSources` as well
- [x] Update the `_attribute_row.html.erb` to not use the `zipped_value`
- [x] ~(Future Task?) TBD (Spot any new file that may use `fetch` or `zipped_values` to display)~

### CODE REVIEW

**Comment by lamtu1** [comment]
fixes #2634

File: app/models/concerns/oregon_digital/metadata_download.rb [hunk 1]
```diff
@@ -33,8 +33,15 @@ def from_local_authority(service, values)
       values.map { |prop| service.label prop }.join('|')
     end
 
-    def from_controlled_field(values)
-      values.map { |prop| controlled_property_to_csv_value(prop).to_s }.join('|')
+    def from_controlled_field(label)
+      # FETCH: Get the SolrDocument that store the label$uri
+      cv_values = SolrDocument.find(id)
```
Comment (straleyb) [inline app/models/concerns/oregon_digital/metadata_download.rb:38]: Same thing here

File: app/views/hyrax/base/_attribute_rows.html.erb [hunk 2]
```diff
@@ -1,7 +1,13 @@
+<%# FETCH: Fetch the label$uri by finding it in SolrDocument %>
+<% cv_label_uris = SolrDocument.find(presenter.id) %>
```
Comment (straleyb) [inline app/views/hyrax/base/_attribute_rows.html.erb:2]: Here too.

File: app/views/hyrax/base/_data_sources.html.erb [hunk 3]
```diff
@@ -8,14 +8,15 @@
     </tr>
   </thead>
   <tbody>
+  <% cv_label_uris = SolrDocument.find(presenter.id) %>
```
Comment (straleyb) [inline app/views/hyrax/base/_data_sources.html.erb:11]: Same thing

**Comment by straleyb** [comment]
My comments aren't deal breakers. Just something I noticed while reading to understand the code.

**Comment by lamtu1** [comment]
> My comments aren't deal breakers. Just something I noticed while reading to understand the code.

Yeah I totally understand that is misleading on what they do. I will double check and rename them to make it better and easier to understand the variables name

**Review by straleyb — APPROVED** [review]

**Pull request merged**